sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -492,6 +493,28 @@ private void setReadinessCheck(
         }
     }
 
+    private static void setTaskKillGracePeriod(
+            Protos.TaskInfo.Builder taskInfoBuilder,
+            TaskSpec taskSpec) throws InvalidRequirementException {
+        Integer taskKillGracePeriodSecondsOptional = taskSpec.getTaskKillGracePeriodSeconds();
```

Comment:
Why is this called `Optional`?  Usually we name things that are of the type `Optional<>` with that suffix.